plugin/Gui/AdminPageAddressListSettings.php

```diff
@@ -366,7 +380,7 @@ public function doExtraEnqueues()
 	protected function checkFixedValues($row)
 	{
 		$table = RecordManager::TABLENAME_LIST_VIEW_ADDRESS;
-		$result = isset($row[$table]['name']) && $row[$table]['name'] != null;
+		$result = isset($row[$table]['name']) && $row[$table]['name'] != null && !empty(trim($row[$table]['name']));
```

Comment:
$row[$table]['name'] != null 
Is this necessary or is not null already checked with isset?

Same question for other classes below.

Reply:
@andernath I will check and let you know soon.

Reply:
@andernath In this case, the isset function already checks undefined and null => so $row[$table]['name'] != null is not needed.
I will check and delete it